README.md

```diff
@@ -13,6 +13,7 @@ Here is a talk from MongoDB Live 2020 about the Community Operator:
 
 - [Documentation](#documentation)
 - [Supported Features](#supported-features)
+  - [Planned Features](#planned-features)
```

Comment:
Not sure about the markup here. Should the line be indented or inline with the lines above and below?

Reply:
it's indented because it's an H3 -- others are H2.